WalletWasabi.Gui/Tabs/AboutViewModel.cs

```diff
@@ -22,16 +22,11 @@ namespace WalletWasabi.Gui.Tabs
 	internal class AboutViewModel : WasabiDocumentTabViewModel
 	{
 		private string _currentBackendMajorVersion;
+		private UpdateChecker _updateChecker;
 
 		public AboutViewModel() : base("About")
```

Comment:
Would something break if `HostedServices` were passed via constructor? It would eliminate need for `var global = Locator.Current.GetService<Global>();`. My Visual studio shows, there's only a few places where `new AboutViewModel()` is called.

Reply:
Removed the whole thing.